run.sh

```diff
@@ -91,7 +91,12 @@ case "$deployment_action" in
             echo "Please enter compute node host name"
             exit 1
         fi
-        remove_compute $2
+        read -r -p "WARNING: Please remove all VM(s) from compute host. Are you sure to continue? [y/n] " user_input
```

Comment:
->Please remove all VM(s) from the compute host.

Reply:
fixed